### PR TITLE
build: Release v18.0

### PR DESCRIPTION
**Pull request by rbradford**
Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -148,6 +148,80 @@
...
+Many thanks to everyone who has contributed to our release:
+
+* Alyssa Ross <hi@alyssa.is>
+* Anatol Belski <anbelski@linux.microsoft.com>
+* Arafatms <arafatms@outlook.com>
+* Bo Chen <chen.bo@intel.com>
+* Fazla Mehrab <akm.fazla.mehrab@vt.edu>
+* Henry Wang <Henry.Wang@arm.com>
+* Jianyong Wu <jianyong.wu@arm.com>
+* Jianyong Wu <Jianyong.Wu@arm.com>
```
Comment (liuw) [inline release-notes.md:217]: Duplicate there.

**Review by liuw — APPROVED** [review]

**Review by likebreath — APPROVED** [review]
Very nice write-up. Only two nits. Also the clippy issue should be fixed by #3095.

**Review by bryteise — APPROVED** [review]

**Review by MrXinWang — APPROVED** [review]
Would be good to have a rebase on top of https://github.com/cloud-hypervisor/cloud-hypervisor/pull/3095 to avoid the clippy issue. Thanks!

File: release-notes.md [hunk 2]
```diff
@@ -148,6 +148,80 @@
...
+### AArch64 for support `virtio-mem`
+
+The reference Linux kernel we recommend for using with Cloud Hypervisor now supports `virtio-mem` on AArch64.
+
+### Live migration on MSHV hypervisor
+
+Live migration is now supported when running on the MSHV hypervisor including
+efficient tracking of dirty pages.
+
+### AArch CPU topology support
```
Comment (MrXinWang) [inline release-notes.md:187]: typo: AArch64

**Review by MrXinWang — APPROVED** [review]

**Review by michael2012z — APPROVED** [review]
Nice writing.

**Pull request merged**